iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/scopes/scopeslist/scopes.scopeslist.component.html

```diff
@@ -0,0 +1,67 @@
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!-- Current Active System Scopes Box -->
+<div class="box box-primary" id="access-tokens-list">
+    <div class="box-header with-border">
+        <div class="well well-small">
+            <button class="btn btn-small refresh-table"><i class="fa fa-refresh"></i> <span data-i18n="common.refresh">Refresh</span></button> &nbsp; 
+            <button class="btn btn-success" id="add-scope-btn" ng-click="$ctrl.openAddScopeDialog()"><i
+                class="fa fa-plus"></i> New Scope</button>
+        </div>
+        <table id="scope-table" class="table table-hover table-striped">
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>Scope</th>
+                    <th></th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr ng-repeat="scope in $ctrl.scopes">
+                    <td class="align-middle" ng-if="scope.defaultScope == true">
+                        <i class="fa fa-flag" title="This Scope is automatically assigned to newly registered clients."></i>
+                    </td>
+                    <td class="align-middle" ng-if="scope.restricted == true">
+                        <i class="fa fa-ban" title="This Scope can be used only by administrators. It is not available for dynamic registration."></i>
+                    </td>
+                    <td class="align-middle" ng-if="scope.restricted != true && scope.defaultScope != true">
+                        <i class="fa fa-flag-o" title="This Scope is not restricted."></i>
+                    </td>
+                    <td class="align-middle">
+                        <h5><a ui-sref="value({id: scope.value})" class="label label-primary"><i class="fa fa-{{scope.icon}}"></i>&nbsp;{{scope.value}}</a></h5>
+                        <small>{{scope.description}}</small>
+                    </td>
+                    <td class="text-right">
+                        <button id="edit_scope_{{scope.id}}" title="Edit Scope" class="btn btn-success btn-xs" ng-click="$ctrl.openEditScopeDialog(scope)">
+                            <i class="fa fa-edit"></i>
+                        </button>
+                        <button title="Delete scope" id="delete_scope_{{scope.id}}" class="btn btn-danger btn-xs" ng-click="$ctrl.openDeleteScopeDialog(scope)">
+                            <i class="fa fa-times"></i>
+                        </button>
+                    </td>
+                </tr>
+            </tbody>
```

Comment:
I tried to add a new scope with both default and restricted flag enabled. This is what I saw after:
![Schermata 2020-09-10 alle 17 28 53](https://user-images.githubusercontent.com/1914039/92754644-575df300-f38b-11ea-9e49-0846e3589209.png)
I think there's a bug